lib/document-builder.ts

```diff
@@ -60,14 +60,16 @@ export class DocumentBuilder {
     description: string = '',
     externalDocs?: ExternalDocumentationObject
   ): this {
-    this.document.tags = this.document.tags.concat(pickBy(
-      {
-        name,
-        description,
-        externalDocs
-      },
-      negate(isUndefined)
-    ) as TagObject);
+    this.document.tags = this.document.tags.concat(
+      pickBy(
+        {
+          name,
+          description,
+          externalDocs
+        },
+        negate(isUndefined)
+      ) as TagObject
+    );
```

Comment:
Can we create a separate PR for this? :)

Reply:
Oh that’s a “npm run format” but yes, of course we can.

Reply:
@kamilmysliwiec do you still want this formatting in a separate PR?

Reply:
removed formatting change